pom.xml

```diff
@@ -7,11 +7,11 @@
         <artifactId>oss-parent</artifactId>
         <version>7</version>
     </parent>
-    
+
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>prettify</artifactId>
-    <version>4-Mar-2014-SNAPSHOT</version>
+    <version>4-Mar-2013</version>
```

Comment:
This should be `4-Mar-2013-1-SNAPSHOT` since there is already a release for `4-Mar-2013` and since Maven does the actual release.